articles/cli.md

```diff
@@ -0,0 +1,178 @@
+## Bonsai Command-Line Interface (CLI)
+
+Bonsai CLI (Command-Line Interface) allows workflows to be launched from the OS command-line tool. In addition to opening workflows with the default graphic interface, the CLI opens the possibility to use other advanced features, that will be covered ahead, such as "no-editor" mode, initialization of variable's values, and specifying layouts.
+
+
+&nbsp;
+___
+### Installation and basic use
+Bonsai CLI can be called by targeting ```Bonsai.exe``` executable. To make sure you are able to call ```Bonsai``` from the command line:
+
+```cmd
+> pathToBonsaiExe\Bonsai.exe
+```
+Alternatively, by adding ```pathToBonsaiExe``` to your System Path, simply:
+
+```cmd
+> Bonsai
+```
+
+This command will open the default Bonsai editor.
+
+___
+&nbsp;
+### Opening a Workflow
+
+To open an existing workflow, ```myWorkflow.bonsai```, use the syntax:
+
+```cmd
+> Bonsai myWorkflow.bonsai
+```
+
+##### Example:
+
+```cmd
+> Bonsai myCameraWorkflow.bonsai
+> Bonsai "C:\Users\User\Desktop\Example\myCameraWorkflow.bonsai"
+```
+___
+&nbsp;
+### Starting a Workflow
+
+Adding additional flags will change the behavior of Bonsai CLI output. For instance, to launch and start a workflow from the command line we add the ```--start``` flag to the previous command:
+
+```cmd
+> Bonsai myWorkflow.bonsai --start
+```
+
+##### Example:
+```cmd
+> Bonsai myCameraWorkflow.bonsai --start
+```
+
+___
+&nbsp;
+### Starting a Workflow without the editor
+
+
+For some applications, such as batch processing, the user might not be interested in having access to the UI. This can be achieved using the ```--no-editor``` flag. Note that the ```--start``` flag behavior is implicit when running with ```--no-editor``` mode.
+
+```cmd
+> Bonsai myWorkflow.bonsai --no-editor
+```
+
+Upon successful launch of your workflow, a Bonsai icon will appear on your system tray that can be used to stop the workflow or open other visualizers.
+
+(![SystemTray Example](~/images/Article_CLI_SystemTrayBonsai.png)
+
+It is worth noting that only assigned visualizers will be available on this list. To assign a visualizer:
+- Open the workflow in editor mode;
+- Right click on top of the node of interest;
+- "Show Visualizer" ➡ Select a visualizer other than "None"
```

Comment:
Edit for clarity: 

"Under menu item "Show Visualizer", select any visualizer other than "None" "